esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -2124,18 +2124,22 @@ unsigned WsWuInfo::getResourceURLCount()
     return 0;
 }
 
+static bool logInitWULogReader = true;
 void WsWuInfo::initWULogReader()
 {
-    LOG(MCdebugProgress,"WsWuInfo loading remote log access plug-in...");
+    if (logInitWULogReader)
+        LOG(MCdebugProgress,"WsWuInfo loading remote log access plug-in...");
```

Comment:
This means that if the configuration changes (e.g. on a helm update) it may log that it failed, but subsequently succeed (silently) or the reverse..

I think it would be better to either remove the tracing altogether or add the logic inside the singleton method in jlog, but also spot when the config changes and update via a ConfigUpdateHook (example use case in eclccserver.cpp, line 1099)

Reply:
@jakesmith ConfigUpdateHook looks to be very useful, i think I've mentioned this before but these types of features need to be publicized loudly to the entire development team!

Reply:
It seems that the reloadConfigHook.installOnce(std::bind(&CWsWorkunitsEx::initWULogReader, this), false); does not run the  initWULogReader() itself.

Reply:
@wangkx I see no issue keeping the logging in the init method if only called at load/reconfig time